app/sidekiq/lighthouse/failure_notification.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class Lighthouse::FailureNotification
+  include Sidekiq::Job
+
+  NOTIFY_SETTINGS = Settings.vanotify.services.benefits_management_tools
+  MAILER_TEMPLATE_ID = NOTIFY_SETTINGS.template_id.evidence_submission_failure_email
+
+  # retry for one day
+  sidekiq_options retry: 14, queue: 'low'
+  # Set minimum retry time to ~1 hour
+  sidekiq_retry_in do |count, _exception|
+    rand(3600..3660) if count < 9
+  end
+
+  sidekiq_retries_exhausted do
+    ::Rails.logger.info('Lighthouse::FailureNotification email could not be sent')
+  end
+
+  def self.notify_client
```

Comment:
Don't think you need `self` here

Reply:
@jerekshoe good catch, I updated it for EVSS but not Lighthouse - doh!